specification/azurefleet/AzureFleet.Management/tspconfig.yaml

```diff
@@ -51,3 +51,16 @@ options:
     head-as-boolean: true
     inject-spans: true
     remove-unreferenced-types: true
+  "@azure-tools/typespec-ts":
```

Comment:
Note: Create or update emitter options here, SDK automation pipeline will automatically detect and generate JS SDK for us.